src/cx16-display-text.h

```diff
@@ -77,10 +78,10 @@ const char* display_smc_unsupported_rom_text[7] = {
     "There is an issue with the CX16 SMC or ROM flash versions.",
     "",
     "Both the SMC and the main ROM must be updated together,",
-    "to avoid possible conflicts of firmware, bricking your CX16.",
+    "to avoid possible conflicts, risking bricking your CX16.",
     "",
-    "The SMC.BIN does not support the current ROM.BIN file",
-    "placed on your SDcard. Upgrade the CX16 upon your own risk!"
+    "The SMC.BIN and ROM.BIN found on your SDCard may not be",
```

Comment:
We only need to check if the SMC.BIN and ROM.BIN on the main board are compatible. I think those that are placed on the SDCard don't really matter.